twoliter/Cargo.toml

```diff
@@ -17,25 +17,26 @@ clap = { version = "4", features = ["derive", "env", "std"] }
 env_logger = "0.11"
 filetime = "0.2"
 flate2 = "1"
-futures= "0.3"
+futures = "0.3"
 hex = "0.4"
 log = "0.4"
-non-empty-string = { version = "0.2", features = [ "serde" ] }
+non-empty-string = { version = "0.2", features = ["serde"] }
+semver = { version = "1", features = ["serde"] }
```

Comment:
Please turn off auto-formatting for TOML files. I don't expect all of our IDEs to agree on the right formatting and we don't want this churn in our git history.